app/assets/javascripts/markets.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Feel free to delete any unused/empty Rails generated files like this one. I usually forgot to, but would get recommended it in code reviews too :)